src/hooks/post-detail/usePostEdit.ts

```
@@ -52,16 +66,16 @@ export function usePostEdit(postInfo: PostDetailUnion | null, refresh: (id: stri
-        (updated as any).hashtags = hashtagsForUi;
-      }
-
       setIsEditingPost(false);
       setEditingPostContent("");
       setEditingImages([]);
       setNewImages([]);
       success("게시물이 수정되었습니다.");
+
+      const updated = await refresh(postInfo.postId);
+      if (updated && updated.__typename === "SnsPost") {
+        (updated as any).hashtags = hashtagsForUi;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)